fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/observer/FedoraEvent.java

```diff
@@ -27,14 +28,12 @@
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
-import java.util.function.Supplier;
 
 import javax.jcr.RepositoryException;
 import javax.jcr.observation.Event;
 
 import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
 import org.fcrepo.kernel.api.utils.EventType;
-import org.fcrepo.mint.UUIDPathMinter;
 
 import com.google.common.base.Function;
 import com.google.common.base.Joiner;
```

Comment:
These Guava types should be replaced with java 8 types `Function` and `StringJoiner` (or potentially `String::join` or `Collectors::joining`).